test/conformance/tests/sriov_operator.go

```diff
@@ -913,6 +921,50 @@ var _ = Describe("operator", func() {
 				createUnschedulableTestPod(testNode, []string{sriovNetworkName}, resourceName)
 			})
 		})
+
+		Context("NAD update", func() {
+			// 24713
+			It("NAD is updated when SriovNetwork spec/networkNamespace is changed", func() {
+				ns1 := "z1"
+				ns2 := "z2"
+				defer namespaces.WaitForDeletion(clients, ns1, 1*time.Minute)
+				defer namespaces.WaitForDeletion(clients, ns2, 1*time.Minute)
+				err := namespaces.Create(ns1, clients)
+				Expect(err).ToNot(HaveOccurred())
+				err = namespaces.Create(ns2, clients)
+				Expect(err).ToNot(HaveOccurred())
+
+				resourceName := "sriovnic"
+				sriovNetworkName := "sriovnetwork"
+				testNode := sriovInfos.Nodes[0]
+				sriovDevice, err := sriovInfos.FindOneSriovDevice(testNode)
+				Expect(err).ToNot(HaveOccurred())
+				createSriovPolicy(sriovDevice.Name, testNode, 5, resourceName)
```

Comment:
nit: policy is not required to test NAD.

Reply:
Done